README.md

```diff
@@ -189,42 +207,28 @@ scss
 ├── packages/
 ├── settings/
 ├── theme/
-│   ├── _uswds-theme-color.scss
-│   ├── _uswds-theme-components.scss
+│   ├── _uswds-theme.scss
 │   ├── _uswds-theme-custom-styles.scss
-│   ├── _uswds-theme-general.scss
-│   ├── _uswds-theme-spacing.scss
-│   ├── _uswds-theme-typography.scss
-│   ├── _uswds-theme-utilities.scss
 │   ├── styles.scss
 ```
 
-- `_uswds-theme-color.scss`: theme color settings
-- `_uswds-theme-components.scss`: theme component settings
+- `_uswds-theme.scss`: custom theme settings
 - `_uswds-theme-custom-styles.scss`: additional project CSS for customizing or adding to what USWDS provides
-- `_uswds-theme-general.scss`: General theme settings
-- `_uswds-theme-spacing.scss`: Theme spacing settings
-- `_uswds-theme-typography.scss`: Theme typography settings
-- `_uswds-theme-utilities.scss`: Utility class output settings
-- `styles.scss`: The primary Sass file that you'll compile. It collects theme settings, USWDS source files, and custom CSS
+- `styles.scss`: The Sass entry point. This is the primary Sass file that you'll compile. It collects theme settings, USWDS source files, and custom CSS
 
 `styles.scss` looks something like the following code. It adds all the project theme settings, then adds USWDS source, and finally adds your project's custom styles:
 
 ```scss
-@import "uswds-theme-general";
-@import "uswds-theme-typography";
-@import "uswds-theme-spacing";
-@import "uswds-theme-color";
-@import "uswds-theme-utilities";
-@import "uswds";
-@import "uswds-theme-custom-styles";
+@forward "uswds-theme";
+@forward "uswds";
+@forward "uswds-theme-custom-styles";
 ```
 
-**Technical note:** The `@import 'uswds'` statement above needs to reference the `uswds.scss` file in `node_modules/uswds/dist/scss`. The gulpfile in [`uswds-gulp`](https://github.com/uswds/uswds-gulp) is set up to do this automatically, using `gulp-sass` and `includePaths`.
+**Technical note:** The `@forward 'uswds'` statement above needs to reference the `uswds.scss` file in `node_modules/@uswds/uswds/packages`. The gulpfile in [`uswds-compile`](https://github.com/uswds/uswds-gulp) is set up to do this automatically, using `gulp-sass` and `includePaths`.
```

Comment:
Points to uswds-gulp instead of uswds-compile

```suggestion
**Technical note:** The `@forward 'uswds'` statement above needs to reference the `uswds.scss` file in `node_modules/@uswds/uswds/packages`. The gulpfile in [`uswds-compile`](https://github.com/uswds/uswds-compile) is set up to do this automatically, using `gulp-sass` and `includePaths`.
```

Reply:
Fixed!